.stylelintrc.yaml

```diff
@@ -0,0 +1,51 @@
+extends:
+- stylelint-config-standard
+- stylelint-config-sass-guidelines
+- stylelint-config-idiomatic-order
+
+plugins:
+- stylelint-color-format
+- stylelint-scss
+- stylelint-no-indistinguishable-colors
+
+ignoreFiles:
+- kbase-extension/static/**/*.{less,scss,css}
+
+rules:
```

Comment:
Many of these rules have not changed from the previous iteration of the file; there are some new rules that are specific to scss that have been added.